packages/terra-form-input/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-form-input",
+  "main": "lib/FormInput.js",
+  "version": "0.0.0",
+  "description": "Element for building out inputs in a form.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-form-input",
+    "FormInput",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-props-table": "^1.10.1"
```

Comment:
Bump this to `^2.0.0`

Reply:
Done b7fb7ce2ada572d12750eb100d869c1e3cc3137b